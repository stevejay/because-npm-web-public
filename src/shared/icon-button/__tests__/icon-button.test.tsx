import IconButton from "../icon-button";
import React from "react";
import { render } from "react-testing-library";
import { noop } from "lodash";
import { IoIosAdd } from "react-icons/io";

test("displays correctly", async () => {
  const { container } = render(
    <IconButton
      ariaLabel="The label"
      icon={IoIosAdd}
      type="button"
      onClick={noop}
    />
  );
  expect(container).toMatchSnapshot();
});
