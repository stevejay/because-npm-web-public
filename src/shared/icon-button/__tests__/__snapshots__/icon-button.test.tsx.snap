// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`displays correctly 1`] = `
<div>
  <button
    aria-label="The label"
    class="button"
    type="button"
  >
    <svg
      fill="currentColor"
      height="1em"
      stroke="currentColor"
      stroke-width="0"
      viewBox="0 0 512 512"
      width="1em"
    >
      <path
        d="M368.5 240H272v-96.5c0-8.8-7.2-16-16-16s-16 7.2-16 16V240h-96.5c-8.8 0-16 7.2-16 16 0 4.4 1.8 8.4 4.7 11.3 2.9 2.9 6.9 4.7 11.3 4.7H240v96.5c0 4.4 1.8 8.4 4.7 11.3 2.9 2.9 6.9 4.7 11.3 4.7 8.8 0 16-7.2 16-16V272h96.5c8.8 0 16-7.2 16-16s-7.2-16-16-16z"
      />
    </svg>
  </button>
</div>
`;
